### ISSUE TITLE
Internal links fail when node ends with ]

### ISSUE THREAD

**Issue by zogommog** [zim-desktop-wiki/zim-desktop-wiki#950]
If you name a node "some [name]", and then try to reference this node from elsewhere by either drag and drop or manually creating a link, it will be created fine initially and function, but as soon as you navigate away from the node, then return to where you created the link, it will be represented like:   some [name] <- where the last "]" character is not highlighted, and the link is broken.

I've confirmed this happens in Debian Buster's (older) version which I can't recall off the top of my head, and in 0.72.0.

TLDR: Zim seems to hate pages that end with the character "]"

P.S. I think Zim is great!

**Comment by nelas** [comment]
See discussion on #596

**Comment by jaap-karssenberg** [comment]
Duplicate
